docs/index.md

```diff
@@ -44,84 +44,46 @@
 
 # Welcome to Streamlit
 
-[Streamlit](https://streamlit.io) is an open-source Python library that makes it
-easy to build beautiful custom web-apps for machine learning and data science.
+[Streamlit](https://streamlit.io/) is an open-source Python library that makes it easy to create and share beautiful, custom web apps for machine learning and data science.
+In just a few minutes you can build and deploy powerful data apps - so let's get started!
 
-To use it, run `pip install streamlit` from a terminal, then import it in Python, write a couple lines
-of code, and run your script with `streamlit run [filename]`. Streamlit watches
-for changes on each save and updates the app live while you're coding. Code
-runs from top to bottom, always from a clean state, and with no need for
-callbacks. It's a simple and powerful app model that lets you build rich UIs
-incredibly quickly. To learn more about how Streamlit works, see [Main
-concepts](main_concepts.md).
+1. Make sure that you have [Python 3.6](https://www.python.org/downloads/) or greater installed.
```

Comment:
Note for later: I'd like there to be a "Official Streamlit (TM) recommended way to install Python and pip" since figuring out best practices on the python ecosystem is quite a pain, and we have the opportunity to become a real leader in this space